src/test/java/apoc/custom/CypherProceduresClusterTest.java

```diff
@@ -151,4 +146,38 @@ public void shouldRemoveFunctionOnOtherClusterMembers() throws InterruptedExcept
             throw e;
         }
     }
+
+    @Test
+    public void overrideCallStatementWithNewArguments() throws Exception {
```

Comment:
I don't see any `apoc.custom.list` call or equivalent in the tests, shouldn't there be some kind of assertions making sure a deleted procedure or function is not listed anymore?

Reply:
Oh I didn't add it in here because we have other tests to track that feature

Reply:
I see, but the initial issue is about discrepancies between `apoc.custom.list` and the actual runnable procedures/functions, isn't it?
When I first read the test, I could not make the connection between the initial problem and how the test makes sure it does not happen anymore.

Reply:
Yes that tricked also us but the `apoc.custom.list` worked properly because after the remove the user added the procedure again, the problem was the background job triggered by the scheduler that cancelled under-the-hood the procedure because the same was inside the `removed` field in the json stored into the graph properties.
Btw you comment made me think that we should get rid of the `removed` field just to avoid future confusions.
I updated the code base, please look at it and lemme know if you still need the test for `apoc.custom.list`